scripts/www/build-cfg-help.pl

```diff
@@ -351,9 +349,13 @@ ()
     } elsif (/^#/) {
         next;
     } elsif ($_ ne "") {
-        print "NOTICE: line $linecount: unknown line '$_'\n";
+        print "NOTICE: line $.: unknown line '$_'\n";
     }
 }
+my $condition;
+foreach $condition (@ifelse) {
+    print "ERROR: missing ENDIF for '$condition'\n";
```

Comment:
Since we now include the line number in `@ifelse`:

```suggestion
    print "ERROR: missing ENDIF to match $condition\n";
```

It would be even better to split the popped item (which is not a "condition" anymore) and say something like "to match ... on line ...".